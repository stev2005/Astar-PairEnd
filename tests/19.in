GGGTTCAAAATAGTCCTTTTGTCCAAATTACGGTCCCCCCCCCTATAAGTTCGAGTAGGATTGCTTTAAACCCGAGCAGAACCGCTCCCACATTGGGCGGCTTTGTCTATTAAAGCTAGCAACTTTGGAAAGAACGTGCTCCTCCGGCGAGGATCATTGTAGCAGGGGATCCACGGTATCCCTCAACTATTAGAAATGGTCACGCTAAATCCAAGAAAAATTACTAGGGAGAATGCGGCGCGTCGGC
15
GCTCAATAACGTCATGTCTGTCAATACGGGCAAACAAAACCTATTTCTAACTCCTCCATTGATGACTCCTGTGGGGCTCCTGACTTGCTCACCAACCTGGGACCGTTTGAAGGATCTCGGGTATTAAATTGCTAGGTCCGCGATGTCGATAGGACCACCTAACAATGTCTCGCCGAATGAGG
CTTGTCATCAACATTTATCTAGCATAGTGCGTACGTGGGAG
CGGCCGAGCACGCAGTTCACAATGACAGCGCTATAGCAATACCCCACCCCGCGCAGTAAATCTTATCGTGCATGCCGGGTAACGGCACCACAGCAGACAATCAT
CAGAGGAGTGAAGCCGTGAAGTA
TATCAGCGGTGACGGAATAGAGATACTGAACTAAACGCTACCAGTTTTGTCGCCGCGCTGCCGGCGTAAGACTCTGAGCGGGATTGCGAAACCGTTCGTCATGTACCAAGGGATCACGGCTGTACGTGCGTCGCCGGGTGCCACAGCGATT
GGATCTCGGATACAGTTCGATGCTATCCGCAATATAAAGGAGGCGACC
TCCCTAGGGTATTCGAATACCGCTATGAATGTCTAAAGGGCGCCTTCAGCCATGTTCCTCCGAGCCGCTATCGAGGATGGATGTCGGTTCAATCGCGAGCAGGTGACGTTCTGTGTCAGCGTCCAAGGACTCTGACGCAGACCGAACGAATGTAATAGCCTAGGGGACTATTCACCAGTCGGTCTTATGCACGGAAAACTAAGCCTGCGGGAAGGTCGTTATCGGCG
CAGCAAGCAATGGCGTTAGCTCATCCCTGTCTTTAAAGCATAGCGAACCTGAATTCAAATTGGTCTTCCACACTGTAGACCTCGGATGAACTGAGTTCCAGGCTGTGGCGTTCACTAAGAATTG
GGGTTTCCAATTGCACATGAAGACAACTAATGGCCCATGATGTGTATATCAATCCTCGTCGGTATCCAAAATGACCAACTGGTCTACATAGCGCGCGTCGTGGCGTTGAACGGC
CCAGAAAGGCAAATCGTTAACATTAACGGGCATGGAGGGCTGCTCGGAGG
TGATGAAATGGTCACCGTTTGCAGTTATCGAACCTTCTAACGTTGAAATAAGCAAATAAAGACTCATTATTCCTATTCTTCTCGACTTCTATTCTCCGACCTTGGTCGAACCAGAATTTAGTTCAGGGAGTCGGTAGAGGCTACCACATAACAAGAGCGTTGACAAGGAACCACGCCTGAAG
AGTACATAACACCATACACGAGCCTATTAGAAATCCAACACGCGGCGTCTGCCTGATGTAATAATCGTCTTGCACACTTGTCTAAGCAAAA
AAATGGGTGCGTCTTTGTCCACGCTTCTTCTTCTGTGATACCTTATGGGTAGAGTTCAGACCATCGGTTCTAGGTGCCCTCC
CTCCAGTATCAGGTCCGGAGCG
TATATACCGATTACGCATCTCAAGGGGCTCTTCGTATCGCCCAGGGTGGACTCCGTAAATCTGGGGTCTCGAGGGAACGGGAGTCAGCATCGGTAACTCAATCGCTGGCCATCGGGGAATTCCGACGGCAGCTTACTGTATTTAGCTACTTCAATAGCGTCCACAAGTTCTGAGTGTGTCCG
