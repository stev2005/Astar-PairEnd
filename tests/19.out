86
18
49
10
71
22
109
54
53
22
92
39
38
9
90
